website/docs/guides/ttl-index.mdx

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 4
+description: Learn about TTL indexes in FerretDB.
+---
+
+# Time-To-Live (TTL) indexes
+
+import CodeBlock from '@theme/CodeBlock';
+import CreateIndex from '!!raw-loader!./requests/1-create-ttl-index.js';
+import InsertTTLData from '!!raw-loader!./requests/2-insert-ttl-data.js';
+
+TTL (Time-To-Live) indexes are useful when you need documents to expire automatically after a certain time. It's ideal for temporary records, logs, or session data.
```

Comment:
I think we should say "to be deleted automatically" because "expire" is not clear. My initial read would be, "They will be marked as expired, updating some metadata field or something." Generally, indexes are used for faster queries, not to **DELETE DATA**. So, we should expand that a bit in the intro.